File: Day Three/java/After Break/CharacterCount.java

public class CharacterCount  {
    public static void main (String [] args) {
    
    String character = "a111BAcb";

    int countTotal = countingSpecialCharacter (character);

        System.out.println("Special character count is: " +countTotal);
    }
    
    public static int countingSpecialCharacter (String word) {
    String inWord = word.toLowerCase();
    String allCharacters = "abcdefghijklmnopqrstuvwxyz0123456789";

   
    int totalCounter =0;


    for (int count = 0; count < allCharacters.length(); count ++) {
        char characterCheck = allCharacters.charAt(count);
        int appearance =0;
        
        for (int index = 0; index < inWord.length(); index ++) {    
        if(characterCheck  ==inWord.charAt(index)) {
        appearance++;
            }
        }
    if (appearance > 1) {
    totalCounter ++;    
        }
    }

    return totalCounter;
    }
}

File: Day Three/python/After Break/character_count.py


character = "a111BAcb"

def counting_special_character (word):
    inword = word.lower()
    allCharacters = "abcdefghijklmnopqrstuvwxyz0123456789"
    total_counter =0

    for element in allCharacters:
        characterCheck = element;
        appearance =0
        
        for index in inword:   
            if (characterCheck == index):
                appearance +=1
      
        if (appearance > 1):
            total_counter +=1

    return total_counter

print(counting_special_character(character))
  

